src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information.mdx

```diff
@@ -67,7 +67,7 @@ You can also use these other methods to create an infrastructure alert condition
     id="alerts-ui"
     title="Use the alerts UI"
   >
-    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alerts & AI > Alert policies > New alert policy > Create new condition**</DoNotTranslate>, then select <DoNotTranslate>**Infrastructure**</DoNotTranslate> as the product.
+    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alerts > Alert policies > New alert policy > Create new condition**</DoNotTranslate>, then select <DoNotTranslate>**Infrastructure**</DoNotTranslate> as the product.
```

Comment:
@homelessbirds check out the repetition of "Alerts > Alerts."

Reply:
Fixed!